docs/Using-the-dataset-validator.md

```diff
@@ -38,8 +38,12 @@ optional arguments:
   -n, --no_portal_checks
                         Skip tests requiring information from the cBioPortal
                         installation
-  -P PORTAL_PROPERTIES, --portal_properties PORTAL_PROPERTIES
-                        portal.properties file path (default: assumed hg19)
+  -species SPECIES, --species SPECIES
+                        species information (default: assumed human)
+  -genome REFERENCE_GENOME, --reference_genome REFERENCE_GENOME
+                        reference genome build (default: assumed hg19)
+  -build GENOME_BUILD, --genome_build GENOME_BUILD
+                        reference genome build (default: assumed 37 for reference genome hg19)
```

Comment:
we should allow to have multiple genome builds for the same genome. They are totally different concepts. If we need to choose one or other, I would vote for use **_genome build_**.

Reply:
but the combination genome=`hg19` and build=`38` is wrong, right?

Reply:
the validator script verifies the combination of genome name and build.

Reply:
@pieterlukasse , I would suggest keep the way it is now till we humanized those vocabularies in the mutation_event table. And, then we complete get rid of the name of genome as well. Let's leave this for a future PR.

Reply:
@pieterlukasse , agree, changed to 
```
--ucsc_build_name
--ncbi_build_name
```
since --grch_build_number does not work for mouse. also users are now only required to supply either one.